03-lection1/01-button/button.css

```diff
@@ -1 +1,57 @@
 /* Стилизация кнопки */
+
+.container {
+  margin-bottom: 10px;
+}
+
+.button {
+  width: 189px;
+  height: 48px;
+  border-radius: 8px;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  color: #ffffff;
+  cursor: pointer;
+  border: 1px solid transparent;
```

Comment:
Может тогда вообще убрать бордер?) 
Можно убрать с помощью 
`border: none;`

а для кнопки alternate - вернуть) тогда получится меньше строк кода с `border-color`)

Reply:
<img width="644" alt="Снимок экрана 2021-01-21 в 10 02 16" src="https://user-images.githubusercontent.com/45252809/105315005-c0723480-5bcf-11eb-8282-0f9ccd56218e.png">
мне не очень нравится без прозрачного бордера, если честно) как лучше сделать всё же?)

Reply:
Давай совсем уберем бордеры с помощью

```suggestion
  border: none;
```
и добавим только для кнопки alternate, потому что там нужен бордер

Reply:
Сделано! с border-none круто получилось, спасибо)

Reply:
🔥